docs/inSkillPurchases.rst

```diff
@@ -0,0 +1,82 @@
+.. _inSkillPurchases:
+
+In-Skill Purchases Reference
+============================
+
+The `in-skill purchasing <https://developer.amazon.com/docs/in-skill-purchase/isp-overview.html>`_ feature enables you to sell premium content such as game features and interactive stories for use in skills with a custom interaction model.
+
+Buying these products in a skill is seamless to a user. They may ask to shop products, buy products by name, or agree to purchase suggestions you make while they interact with a skill. Customers pay for products using the payment options associated with their Amazon account.
+
+For more information about setting up ISP with the ASK CLI follow this `link <https://developer.amazon.com/docs/in-skill-purchase/use-the-cli-to-manage-in-skill-products.html>`_. And to understand what's the process behind the ISP requests and responses to the Alexa Service click `here <https://developer.amazon.com/docs/in-skill-purchase/add-isps-to-a-skill.html>`_.
+
+With Voxa, you can implement all ISP features like buying, refunding and upselling an item:
+
+.. code-block:: javascript
+
+  skill.onIntent('BuyIntent', async (voxaEvent) => {
```

Comment:
I know ISP is an Alexa feature, but shouldn't this be `app` instead of `skill`?

Reply:
Same for all other `skill` statements

Reply:
Yeah, I think it makes sense. I was just following the same structure of the docs. If you look at other docs' examples, it uses `skill` as well. Also, in the headspace action and skill, the guys are still using `skill` for their intents/states:
https://github.com/mediarain/headspace-skill/blob/V3-staging/app/states/alexa.states.js

@armonge thoughts on this change ?? If we go with @lvelasquezm suggestion, I'd have to change all the docs (and probably the tests?)

I also think is a matter of the developer's preference. I really like `skill` since I'm coming from the voxa2 version and I'm used to that. But open to any guidance on this topic.

Reply:
We should call them `app`, in Voxa3 most of the code is part of the VoxaApp, the `skill` is just the actual wrapper. Also, if you see the docs calling stuff `skill` instead of `app` go ahead and change that, wherever it makes sens